packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -23,9 +23,16 @@ import { DEFAULT_FUNCTIONS } from './functions'
 import { BotonicPluginFlowBuilderOptions, KnowledgeBaseResponse } from './types'
 import { resolveGetAccessToken } from './utils'
 
+function getFlowUrl(request: PluginPreRequest) {
+  const baseFlowUrl = process.env.FLOWBUILDER_API_URL
+  return request.session.isStagingIntegration
+  ? `${baseFlowUrl}/latest`
+  : `${baseFlowUrl}/draft`
```

Comment:
Shouldn't this be the other way around? `draft` version when `isStagingIntegration` is `true` and `latest` otherwise?

Reply:
yes